alf/environments/carla_sensors.py

```diff
@@ -955,6 +957,15 @@ def _get_traffic_light_waypoints(self, traffic_light):
                     wpx = next_wp
                 else:
                     break
+            if wpx.transform.location.distance(
+                    area_loc) >= self.RED_LIGHT_ENFORCE_DISTANCE - 2:
```

Comment:
What is special about -2 here?

Reply:
distance between the center of the car and its tail